pkg/mqtrigger/kafka/Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG GO_VERSION=1.14
+
+FROM ubuntu:18.04 AS base
+WORKDIR /
+RUN apt update && apt install -y ca-certificates && rm -rf /var/lib/apt/lists/*
+
+FROM golang:${GO_VERSION} AS builder
```

Comment:
There is a small improvement that can be done here - the builder and final image is same. Please check https://github.com/fission/fission/blob/master/environments/go/Dockerfile and try to optimize this one

Reply:
Done